libethash-cl/CLMiner.cpp

```diff
@@ -20,7 +20,7 @@ namespace eth
 unsigned CLMiner::s_workgroupSize = CLMiner::c_defaultLocalWorkSize;
 unsigned CLMiner::s_initialGlobalWorkSize = CLMiner::c_defaultGlobalWorkSizeMultiplier * CLMiner::c_defaultLocalWorkSize;
 
-constexpr size_t c_maxSearchResults = 255;
+constexpr size_t c_maxSearchResults = 15;
```

Comment:
This should also be `static`.

Reply:
It could be, but it's not a variable... it is a hard constant.